### PR TITLE
Added ids to footer sections

### PR DESCRIPTION
**Pull request by faloi**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -90,24 +90,24 @@
...
-    <div class="row">
+    <div id="footer-copyright" class="row">
       <div class="col-md-4 text-left">
         <p>&copy; Copyright 2015-<%= DateTime.now.year %>
           <a href="http://mumuki.org/" class="mu-org-link"><span class="da da-mumuki-circle"></span> Mumuki Project</a>
         </p>
       </div>
 
-      <div class="col-md-4 text-center">
+      <div id="footer-auth0" class="col-md-4 text-center">
```
Comment (flbulgarelli) [inline app/views/layouts/application.html.erb]: It is not necessary auth0, it is provider-dependent. In fact, when using SAML, no content is displayed within this div.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -90,24 +90,24 @@
   <div class="container">
     <hr>
 
-    <div class="row">
+    <div id="footer-authoring" class="row">
```
Comment (flbulgarelli) [inline app/views/layouts/application.html.erb]: It is ok to provide a hook for customizing the authoring. However, please notice that as [Share Alike](https://creativecommons.org/licenses/by-sa/2.5/) License states, you are obliged to display who is the author.

Comment (flbulgarelli) [inline app/views/layouts/application.html.erb]: That means, you are free to do whatever you want with your customizations, and there is nothing wrong with adding the ID. But this is a just a friendly warning about not doing bad things :stuck_out_tongue:

**Comment by faloi** [comment]
@flbulgarelli please review again.

**Pull request merged**